### ISSUE TITLE
2.5.4 Auto-Type: macOS Mojave keeps prompting for Accessibility Options permissions after already granted

### ISSUE THREAD

**Issue by iipamonkey** [keepassxreboot/keepassxc#4788]
## Overview
[TIP]:  # ( DO NOT include screenshots of your actual database! )
[NOTE]: # ( Give a BRIEF summary about your problem )
After updating to 2.5.4 from 2.5.3, auto-type no longer works. Keeps prompting for accessibility options permissions even though already granted.

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. Select entry
2. Perform auto-type shortcut
3. Observe prompt

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
Normal auto-type behavior from all previous versions.

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )
Auto-type fails and keeps prompting for accessibility options permissions after it is already granted.

## Context
[NOTE]: # ( Give us any additional information you may have. )
Prompt mentions "Screen Recording" permissions required; there is no such permission in macoOS Mojave.

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - Version 2.5.4
Revision: dcca5aa

Qt 5.14.1
Debugging mode is disabled.

Operating system: macOS Mojave (10.14)
CPU architecture: x86_64
Kernel: darwin 18.7.0

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey
- TouchID

Cryptographic libraries:
 libgcrypt 1.8.5

Operating System: macOS Mojave 10.14.6

**Comment by iipamonkey** [comment]
<img width="420" alt="Screen Shot 2020-05-28 at 2 26 33 PM" src="https://user-images.githubusercontent.com/20322778/83184669-6330ec80-a0ef-11ea-81f5-5870d8ed6854.png">

**Comment by droidmonkey** [comment]
You need to restart your computer. This is discussed elsewhere on the issue board. This is a mac issue.
